com.unity.sg2/Editor/ContextLayeredDataStorage/Element.cs

```diff
@@ -61,6 +62,18 @@ internal virtual SerializedElementData ToSerializedFormat()
             return new SerializedElementData(ID, Header.GetType().AssemblyQualifiedName, Header.ToJson(), null, null);
         }
 
+        public virtual Element MakeCopy()
```

Comment:
Should this be a `Clone` method override?

Reply:
Oh is that a c# convention I should follow? I wasnt familiar with it, but I can make that change if you think I should

Reply:
It appears that the convention is to have copy constructors. I'm not sure if we should standardize on this. We can take this to a Slack thread and I will resolve this convo.